pkg/rancher-components/src/components/Form/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from './Checkbox';
```

Comment:
If we are introducing barrel exports, we should probably inform it in the PR notes or maybe mention it in the documentation.
Could you also maybe explain why is this needed?

Reply:
See my comment https://github.com/rancher/dashboard/pull/5504#discussion_r876381466